amf-shapes/shared/src/main/scala/amf/shapes/client/platform/model/domain/jsonldinstance/JsonLDScalar.scala

```diff
@@ -0,0 +1,18 @@
+package amf.shapes.client.platform.model.domain.jsonldinstance
+
+import amf.core.client.scala.model.domain.AmfScalar
```

Comment:
This should be platform not scala. Looks like it does not exists, we should create it.